content/md/first-steps/manage-locales.md

```diff
@@ -26,3 +26,15 @@ A locale is activated when it is added to a channel, it cannot be enabled from t
 :::info
 Locales which are not used in at least one channel are shown as inactive.
 :::
+
+# How to add a new locale?
+
+You are missing a locale? It is very simple to add a new locale using imports.
+For instance, to add a 'en_HK' locale, follow these steps:
+1.  Create an import file XLSX or CSV containing two values: a locale `code` and an `activated` column:
+`code;activated`
+`en_HK;0`
+1.  Import this file using an import job (csv_locale_import or xlsx_locale_import). Once the file is imported, the new locale will appear under Settings/Locales.
```

Comment:
Put `csv_locale_import` between `` things.
Idem for  `xlsx_locale_import`.
Idem for `Settings`/`Locales`